### DISCUSSION TITLE
Information about active widgets

### DISCUSSION THREAD

**Discussion by 1000camels** [elementor/elementor#15078] [Feature Request]
As more and more widgets are available for Elementor, it would be great if there was a way to know what plugin or developer produced a given widget. This information is buried and not easy to access with a widget on a page. I am imagining some place in the widget settings that would simply display this information.

It would also be interesting to be able to audit widgets in use, so you could know whether a given widget was being used.

Darcy

**Comment by aaliyahfraser** [comment]
I have also posted a blog on this topic. This is because Elementor has so many free amazing add-ons. Easy-to-use features made creating websites a breeze. Here we talk about “Best Elementor Addons and Plugins for WordPress 2023 – Compared” You can check out it at – https://quomodosoft.com/best-elementor-addons-and-plugins/

**Reply by Vossha** [reply]
> I have also posted a blog on this topic. This is because Elementor has so many free amazing add-ons. Easy-to-use features made creating websites a breeze. Here we talk about “Best Elementor Addons and Plugins for WordPress 2023 – Compared” You can check out it at – https://quomodosoft.com/best-elementor-addons-and-plugins/

@avivu @nicholaszein Spam/link farming 🙃 See also this newly opened discussion: ([https://github.com/elementor/elementor/discussions/20904](https://github.com/elementor/elementor/discussions/20904))
